outputs.tf

```diff
@@ -1,6 +1,6 @@
 output "public_ip" {
   description = "Public IP of instance (or EIP )"
-  value       = "${coalesce(join("", aws_eip.default.*.public_ip), aws_instance.default.public_ip)}"
+  value       = "${coalescelist(aws_eip.default.*.public_ip, aws_instance.default.*.public_ip)}"
```

Comment:
since `${coalescelist(aws_eip.default.*.public_ip, aws_instance.default.*.public_ip)}` returns a list,
it should be `${join("", coalescelist(aws_eip.default.*.public_ip, aws_instance.default.*.public_ip))}`
to return one item from the list

Reply:
doesn't make a sense, in the that output can be only one element or nothing

Reply:
the output returns one IP (as the description says).
not a list